core/storage/audit/gae_models_test.py

```diff
@@ -57,6 +57,27 @@ def test_get_deletion_policy(self) -> None:
             audit_models.RoleQueryAuditModel.get_deletion_policy(),
             base_models.DELETION_POLICY.KEEP)
 
+    def test_get_export_policy(self) -> None:
+        export_policy = audit_models.RoleQueryAuditModel.get_export_policy()
+        self.assertEqual(
+            export_policy['user_id'],
+            base_models.EXPORT_POLICY.NOT_APPLICABLE)
+        self.assertEqual(
+            export_policy['intent'],
+            base_models.EXPORT_POLICY.NOT_APPLICABLE)
+        self.assertEqual(
+            export_policy['role'],
+            base_models.EXPORT_POLICY.NOT_APPLICABLE)
+        self.assertEqual(
+            export_policy['username'],
+            base_models.EXPORT_POLICY.NOT_APPLICABLE)
```

Comment:
I think it would be clearer to construct the full expected export policy dict and then compare that dict with `export_policy`. Right now it's not obvious what the full dict should look like. This test would also still pass if `export_policy` had extra keys, which we don't want

Reply:
Thanks @U8NWXD. Actually, I did create a `sample_dict = {'title': base_models.EXPORT_POLICY.NOT_APPLICABLE ....}` like dict and compared it to `.get_export_policy()`. However, the terminal returned failure.  Thus, I shifted to this way trying to remove the issues related to extra fields. 


![Screenshot from 2022-01-07 23-31-35](https://user-images.githubusercontent.com/76519771/148633212-7b1961a3-2d7f-4f29-9941-4cff4bbcb33d.png)

Reply:
Those "extra" fields are probably coming from the superclass `get_export_policy()`:

https://github.com/oppia/oppia/blob/f74b606e8511cd4296b3c99aad37e53b66cca196/core/storage/audit/gae_models.py#L80

You can look in the superclass for the fields you need to add

Reply:
Got it! Am on it! Thanks @U8NWXD